src/core/solvers/ddp.cpp

```diff
@@ -324,24 +340,31 @@ void SolverDDP::forwardPass(const double steplength) {
   if (raiseIfNaN(cost_try_)) {
     throw_pretty("forward_error");
   }
+  STOP_PROFILER("SolverDDP::forwardPass");
 }
 
 void SolverDDP::computeGains(const std::size_t t) {
+  START_PROFILER("SolverDDP::computeGains");
```

Comment:
You also need to include this code for BoxDDP::computeGains

Reply:
Same comment as above

Reply:
We re-implement this function in `BoxDDP`. See this code: https://github.com/loco-3d/crocoddyl/blob/devel/src/core/solvers/box-ddp.cpp#L47-L78

It is the same for the `BoxFDDP` as you can see here: https://github.com/loco-3d/crocoddyl/blob/devel/src/core/solvers/box-fddp.cpp#L47-L78